### PR TITLE
Updating docs to include updated actions (except JWT)

### PR DESCRIPTION
**Pull request by KristopherPaulsen**
Updated to be iteratively better than what we have now.

### CODE REVIEW

File: docs/http/traffic-policy/actions/redirect.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ When executed this action will abort the request and replace all matches on the
 | `from`        | string                    | A regular expression string to match inside of the URL. Supports interpolating the results of [cel expressions](#templating-syntax) outside of regular regex.       |
 | `to`          | string                    | A regular expression string to replace the `from` match with. Supports interpolating the results of [cel expressions](#templating-syntax) outside of regular regex. |
 | `status_code` | int                       | 3xx status code used for redirecting, 302 by default.                                                                                                               |
-| `headers`     | Map&lt;string, string&gt; | Headers to be added to the request.                                                                                                                                 |
+| `headers`     | Map&lt;string, string&gt; | Headers to be added to the response.                                                                                                                                |
```
Comment (rkolavo) [inline docs/http/traffic-policy/actions/redirect.mdx:40]: https://github.com/ngrok-private/ngrok/pull/30854

File: examples/actions/url-rewrite.mdx [hunk 2]
```diff
@@ -51,15 +51,15 @@ export const TemplatingExamples = () => (
...
 			config={{
 				actions: [
 					{
 						type: "url-rewrite",
 						config: {
 							format: "ngrok",
-							from: "/v0/api/when-will-then-be-now?not-now=${conn.start_ts > time.now}",
-							to: "/v1/api/soon?now=${conn.start_ts < time.now}",
+							from: "?not-now=${conn.ts.start > time.now}",
+							to: "?now=${conn.ts.start < time.now}",
```
Comment (rkolavo) [inline examples/actions/url-rewrite.mdx:62]: think these vars in docs need to be updated still too

Comment (rkolavo) [reply examples/actions/url-rewrite.mdx:62]: https://github.com/ngrok/ngrok-docs/blob/main/docs/http/traffic-policy/expressions/variables.mdx?plain=1#L15

Comment (rkolavo) [reply examples/actions/url-rewrite.mdx:62]: and here https://github.com/ngrok/ngrok-docs/blob/main/docs/tcp/traffic-policy/expressions/variables.mdx?plain=1#L14

Comment (rkolavo) [reply examples/actions/url-rewrite.mdx:62]: and here https://github.com/ngrok/ngrok-docs/blob/main/docs/tls/traffic-policy/expressions/variables.mdx?plain=1#L14
(these should probably be updated to be component)

Comment (KristopherPaulsen) [reply examples/actions/url-rewrite.mdx:62]: I think Ben is doing these and didn't want to collide with him @benjaminchan Is that still true?

Comment (KristopherPaulsen) [reply examples/actions/url-rewrite.mdx:62]: eh, I'll just do it, #yolo

File: docs/obs/_event-types-tables.mdx [hunk 3]
```diff
@@ -230,7 +230,7 @@ This event type supports filters and selectable fields.
 | conn.server_ip | string | filterable | The IP address of the server that received the request |
 | conn.server_name | string | filterable | The hostname associated with this connection. |
 | conn.server_port | int32 | filterable | The port that the connection for this request came in on |
-| conn.start_ts | timestamp | The timestamp when the TCP connection to the ngrok edge is established |
```
Comment (rkolavo) [inline docs/obs/_event-types-tables.mdx:233]: these fields on v0 events will nottt be changing

File: docs/obs/reference.mdx [hunk 4]
```diff
@@ -95,7 +95,7 @@ Triggers when an HTTP request completes.
...
 | conn.server_port              | int32                                 | filterable                                                                                                                                                                                             | The port that the connection for this request came in on |
-| conn.start_ts                 | timestamp                             | The timestamp when the TCP connection to the ngrok edge is established                                                                                                                                 |                                                          |
+| conn.ts.start                 | timestamp                             | The timestamp when the TCP connection to the ngrok edge is established                                                                                                                                 |                                                          |
```
Comment (rkolavo) [inline docs/obs/reference.mdx:98]: same here, the events aren't changing

**Review by rkolavo — APPROVED** [review]

**Pull request merged**